### ISSUE TITLE
Adapt Accessibility tests: cannot interact with Slider components using only keyboard

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt-contrib-slider#53]
_From @shauncg on June 20, 2014 15:51_

It's not clear how the Slider component is supposed to be interacted with using keyboard navigation. The JAWS descriptive text doesn't offer any hints. 

Pressing the spacebar or enter key (the usual JAWS command/control keys) with the slider lozenge selected just opens the drawer (which is a bit odd - I'm not sure why this occurs). 

_Copied from original issue: adaptlearning/adapt_framework#421_

**Comment by oliverfoster** [comment]
Fixed with accessibility updated
